Dockerfile

```diff
@@ -1,13 +1,33 @@
-FROM ubuntu:16.04
+# Multi-stage docker build
+# Build stage
+FROM golang:1.14 AS builder
 
-ARG TARGETARCH
-ENV EXPORTER=/exporter-${TARGETARCH}
+LABEL maintainer="LitmusChaos"
 
-RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/cache/apk/*
+# LD_FLAGS is passed as argument from Makefile. It will be empty, if no argument passed
+ARG TARGETPLATFORM
 
-COPY .$EXPORTER /
+ADD . /chaos-exporter
+WORKDIR /chaos-exporter
 
-EXPOSE 8080
+RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
+    export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2)
 
-CMD ["sh", "-c","$EXPORTER"]
+RUN go env
```

Comment:
Is this for testing purpose?

Reply:
No, it is good to show the go env because we are having multi-arch support